client/views/author/submissions.list.html

```diff
@@ -20,5 +20,19 @@ <h2 class="panel-title">{{s.title}}</h2>
         </div>
       </div>
     </div>
+    <div ng-show="!submissions || submissions.length == 0">
```

Comment:
Der parent dieses div's hat die Klasse `col-md-8 col-md-offset-2`. Bei den entsprechenden div's in den views `client/views/reviewer/reviews.overview.html` und `client/views/shared/reviews.list.html` hat der parent die Klasse `col-md-10 col-md-offset-1`. Ich denke die gehört hier auch gesetzt.